GVLXamarinFormsExample/GVLXamarinFormsExample.iOS/GVLXamarinFormsExample.iOS.csproj

```diff
@@ -130,10 +137,16 @@
     <Reference Include="Xamarin.iOS" />
     <Reference Include="System.Numerics" />
     <Reference Include="System.Numerics.Vectors" />
+    <Reference Include="NativeLibrary">
+      <HintPath>..\..\NativeLibrary.dll</HintPath>
```

Comment:
Could you add the Xamarin projects to build the Native Library as well? Would be useful to have everything we need in this one repo.

Reply:
It's not streamlined enough at this stage. This will take a little longer.

Reply:
Ok, thanks.